stores/amazon_monitoring.py

```diff
@@ -158,6 +163,11 @@ def __init__(
             )
             self.sessions_list[idx].headers.update({"user-agent": ua.random})
 
+        for idx in range(self.proxies_length):
+            AmazonMonitor.proxies.update({idx: {self.proxies[idx]: time.time()}})
+            log.debug(f"Adding [{self.proxies[idx]}] to the pool")
+        AmazonMonitor.proxies_length = self.proxies_length
+
```

Comment:
This is also very confusing - you are setting something in the class after all the class objects have been created. Maybe this works (again, not an expert at Python/development), but I feel this proxy thing should just be it's own class... or the connector class should be overloaded to deal with what wants to be done, and just pass the same connector to every session.

Reply:
Got it. I'll try to separate it from AmazonMonitor.